game/missiongenerator/aircraft/waypoints/waypointgenerator.py

```diff
@@ -118,7 +119,7 @@ def create_waypoints(self) -> tuple[timedelta, list[FlightWaypoint]]:
 
     def builder_for_waypoint(self, waypoint: FlightWaypoint) -> PydcsWaypointBuilder:
         builders = {
-            FlightWaypointType.DROP_OFF: CargoStopBuilder,
+            FlightWaypointType.AIR_DROP: AirDropBuilder,
```

Comment:
Unused

Reply:
Well. It is at the moment but it was there already and is a preparation for later implementation. I dont see the point of removing it. Way easier for people to continue the work later on if there is preparation work done.

Reply:
> I dont see the point of removing it.

Because unused code is confusing and takes effort to maintain.